### PR TITLE
feat: loggroup handled by user

### LINKED ISSUE #9: Add a parameter for custom log group
Log groups should be handled by users.

### PR DESCRIPTION
**Pull request by guan840912**
feat: loggroup handled by user
To Close #9

### CODE REVIEW

File: src/valheim.ts [hunk 1]
```diff
@@ -87,10 +102,7 @@ export class ValheimWorld extends Construct {
 
     const containerDefinition = taskDefinition.addContainer('ValheimContainer', {
       image: props?.image ?? ecs.ContainerImage.fromRegistry('lloesche/valheim-server'),
-      logging: new ecs.AwsLogDriver({
-        streamPrefix: 'valheim',
-        logRetention: logs.RetentionDays.ONE_DAY,
-      }),
+      logging: new ecs.AwsLogDriver(awsLogDriverProps),
```
[the remark this answers is not in the document]
Comment (guan840912) [reply src/valheim.ts]: I think option 2 is better.

File: src/valheim.ts [hunk 2]
```diff
@@ -51,6 +51,12 @@ export interface ValheimWorldProps {
   readonly environment?: {
     [key: string]: string;
   };
+  /**
+   * valheim server log Group.
```
[the remark this answers is not in the document]
Comment (guan840912) [reply src/valheim.ts]: okay

File: src/valheim.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (gotodeploy) [reply src/valheim.ts]: If logGroup accepts some kinds of log drivers, the type should be "LogDriver".

```ts
readonly logGroup?: ecs.LogDriver;
```

[the remark this answers is not in the document]
Comment (guan840912) [reply src/valheim.ts]: sorry , fixed it .

**Comment by gotodeploy** [comment]
LGTM

**Pull request merged**